CHANGELOG.md

```diff
@@ -74,6 +74,7 @@ The table below shows which release corresponds to each branch, and what date th
 
 ## 5.0.0 (`dev`)
 
+- [#2530][2530] Do NOT error when passing directory arguments in `checksec`.
```

Comment:
You need to add the link for the [2530] name too below this list. Please add new entries to the end of the changelog too!